cleanlab/internal/label_quality_utils.py

```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
 
     Parameters
     ----------
-    pred_probs : np.ndarray (shape (N, K))
+    pred_probs
       Each row of this matrix corresponds to an example x and contains the model-predicted
       probabilities that x belongs to each possible class: P(label=k|x)
 
-    min_allowed_prob : float, default=1e-6
+    min_allowed_prob
       Minimum allowed probability value. Entries of `pred_probs` below this value will be clipped to this value.
       Ensures entropy remains well-behaved even when `pred_probs` contains zeros.
 
     Returns
     -------
-    entropy : np.ndarray (float)
+    entropy : np.ndarray 
```

Comment:
You can also remove the type annotation for this argument in its docstring here. Note that it is automatically added under "Return Type" already.